src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -288,6 +293,30 @@ def check_token_status_array(self, clusters_token_array):
     def check_token_status(self, clustersTokenMap=None):
         clusters_token_map = json.loads(clustersTokenMap)
         return self.check_token_status_array(clusters_token_map)
+    
+    @Endpoint('PUT')
+    @UpdatePermission
+    def set_managed_by_config(self, url: str, fsid: str):
+        managed_by_config = Settings.MANAGED_BY_CLUSTERS
+        if managed_by_config is not None:
+            managed_by_config.append({ 'url': url, 'fsid': fsid})
+        Settings.MANAGED_BY_CLUSTERS = managed_by_config
+        return Settings.MANAGED_BY_CLUSTERS
```

Comment:
have you tried to use the exisiting apis to do this? We have `api/mgr/modules` to do these kinds of thing. if it is just simply appending some configuration, why not use the existing APIs if it works.

Reply:
Updated the PR